app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsScreen.kt

```diff
@@ -113,6 +112,12 @@ private fun GroupConversationDetailsContent(
     val deleteGroupDialogState = rememberVisibilityState<GroupDialogState>()
     val leaveGroupDialogState = rememberVisibilityState<GroupDialogState>()
 
+    val context = LocalContext.current
```

Comment:
Do we need to move it to child composable? 😄

Reply:
doone, moved to parent component ! :D